distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/InstallPluginCommandTests.java

```diff
@@ -393,6 +401,16 @@ public void testSomethingWorks() throws Exception {
         assertPlugin("fake", pluginDir, env.v2());
     }
 
+    public void testMultipleWorks() throws Exception {
```

Comment:
Can we also add tests for the duplicate plugin detection, and failure case wiping the plugins (ie making sure the transactional behavior works)?

Reply:
I pushed tests for this.